MovieGallery/Pages/Search.cshtml

```diff
@@ -13,8 +13,8 @@
                 <div class="col-md-12">
                     <div class="search">
                         <i class="fa fa-search"></i>
-                        <input type="text" class="form-control" placeholder="Enter year to start search..." name="SearchbyYear" id="SearchbyYear">
-                        <button type="submit" class="btn btn-secondary">Search by Year</button>
+                        <input type="text" maxlength="4" pattern="\d*" class="form-control" placeholder="Enter year to start search..." name="SearchbyYear" id="SearchbyYear">
```

Comment:
Added client-side validation on search to accept only numeric entry. The code threw exception when searched using special characters.
https://www.smashingmagazine.com/2009/07/web-form-validation-best-practices-and-tutorials/

Reply:
Huh, nice one.